buildpipeline/tests/Dotnet-CoreClr-Trusted-BuildTests.json

```diff
@@ -0,0 +1,378 @@
+{
+  "build": [
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Run script $(VS140COMNTOOLS)\\VsDevCmd.bat",
+      "timeoutInMinutes": 0,
+      "condition": "ne(variables['PB_SkipTests'], 'true')",
+      "refName": "Task1",
+      "task": {
+        "id": "bfc8bf76-e7ac-4a8c-9a55-a944a9f632fd",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "$(VS140COMNTOOLS)\\VsDevCmd.bat",
+        "arguments": "",
+        "modifyEnvironment": "true",
+        "workingFolder": "",
+        "failOnStandardError": "false"
+      }
+    },
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Run clean.cmd",
+      "timeoutInMinutes": 0,
+      "condition": "ne(variables['PB_SkipTests'], 'true')",
+      "refName": "Task2",
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "clean.cmd",
+        "arguments": "-all",
+        "workingFolder": "",
+        "failOnStandardError": "false"
+      }
+    },
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Run sync.cmd",
+      "timeoutInMinutes": 0,
+      "condition": "ne(variables['PB_SkipTests'], 'true')",
+      "refName": "Task3",
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "sync.cmd",
+        "arguments": "-ab -AzureAccount=$(CloudDropAccountName) -AzureToken=$(CloudDropAccessToken) -Container=$(ParentContainerName) -RuntimeId=$(Rid) -BlobNamePrefix=$(ParentBlobNamePrefix)$(ParentBuildType)/TestNativeBins/$(Rid) -- /p:DownloadDirectory=$(Build.SourcesDirectory)/packages/TestNativeBins/$(Rid)/$(ParentBuildType)",
```

Comment:
Given this is an independent pipeline what is the strategy for finding the right azure blob for the matching coreclr bits?

Reply:
The blob is statically predictable given the input parameters to the build pipeline and the build pipeline's build id.

Reply:
OK. That does mean that each of the these pipelines will need to be kicked off with same input parameters. Do we have any good way to ensure that?